test/black-box/iteminformation/ItemInformationGetCommand.test.ts

```diff
@@ -6,51 +6,65 @@ import { rpc, api } from '../lib/api';
 import { BlackBoxTestUtil } from '../lib/BlackBoxTestUtil';
 import { Commands } from '../../../src/api/commands/CommandEnumType';
 import { CreatableModel } from '../../../src/api/enums/CreatableModel';
+import { GenerateListingItemTemplateParams } from '../../../src/api/requests/params/GenerateListingItemTemplateParams';
 
 describe('ItemInformationGetCommand', () => {
     const testUtil = new BlackBoxTestUtil();
     const method = Commands.ITEMINFORMATION_ROOT.commandName;
     const subCommand = Commands.ITEMINFORMATION_GET.commandName;
 
-    const testDataListingItemTemplate = {
-        profile_id: 0,
-        itemInformation: {
-            title: 'Item Information with Templates First',
-            shortDescription: 'Item short description with Templates First',
-            longDescription: 'Item long description with Templates First',
-            itemCategory: {
-                id: null
-            }
-        }
-    };
+    let createdListingItemTemplate;
     let createdListingItemTemplateId;
+
```

Comment:
no need for both of these, createdListingItemTemplate is enough.
missing type:
let createdListingItemTemplate: resources.ListingItemTemplate;